### ISSUE TITLE
[feature] should output record base on station

### ISSUE THREAD

**Issue by kan01234** [kan01234/ur-web-spider#24]
input
```
JR中央線「高尾」駅徒歩25～30分
JR中央線「高尾」駅バス7分 徒歩1～11分
JR中央線ほか｢八王子｣駅 徒歩13分
```

ouptut
JR中央線「高尾」駅, 
|station|best case walk|worst case walk|best case bus|worst case bus|
|:------|:--------------|----------------|--------------|---------------|
|JR中央線「高尾」駅|25|30|8|19|
|JR中央線ほか｢八王子｣駅|13|13|||

**Comment by kan01234** [comment]
close #19 #18

**Comment by kan01234** [comment]
fixed on #33
